kolena/workflow/test_run.py

```diff
@@ -370,19 +374,24 @@ def process_results(results: Optional[EvaluationResults], config: Optional[Evalu
                     inferences,
                     test_case_test_samples,
                     configuration,
-                )
+                )  # type: ignore
                 process_results(evaluation_results, configuration)
         else:
             test_case_test_samples._set_configuration(None)
-            evaluation_results = evaluator(test_samples, ground_truths, inferences, test_case_test_samples)
+            evaluation_results = evaluator(
+                test_samples,
+                ground_truths,
+                inferences,
+                test_case_test_samples,
+            )  # type: ignore
             process_results(evaluation_results, None)
 
         log.info("uploading test case metrics")
         self._upload_test_case_metrics(test_case_metrics)
         log.info("uploading test case plots")
-        self._upload_test_case_plots(test_case_plots)
+        self._upload_test_case_plots(test_case_plots)  # type: ignore
```

Comment:
Nit: for these 2 ignores, seems like alternative could be

```
    def _upload_test_case_plots(
        self,
        plots: Mapping[int, Mapping[Optional[EvaluatorConfiguration], Optional[List[Plot]]]],
    ) -> None:
      ...

    def _upload_test_suite_metrics(
        self,
        metrics: Mapping[Optional[EvaluatorConfiguration], Optional[MetricsTestSuite]],
    ) -> None:
```

Reply:
Updated, thanks for the suggestion :+1: